MyArrayList.java

```
@@ -0,0 +1,163 @@
+                }
+            } else {
+                T[] newArr = (T[]) new Object[sizeArr];
+                for (int i = 0; i < arr.length; i++) {
+                    newArr[element] = arr[i];
+                    element++;
+                }
+                for (int i = 0; i < list.size(); i++) {
+                    newArr[element] = (T) list.get(i);
+                    element++;
+                }
+                arr = newArr;
+            }
+            size = sizeArr;
+        }
+    }
+
+    @Override
+    public T get(int index) {
+        return arr[index];
+    }
+
+    @Override
+    public void set(T value, int index) {
+        this.arr[index] = value;
+    }
+
+    @Override
+    public T remove(int index) {
+        if (index != (size)) {
+            T[] newArr = (T[]) new Object[size];
+            int element = 0;
+            for (int i = 0; i < this.size; i++) {
+                if (i != index) {
+                    newArr[element] = arr[i];
+                    element++;
+                }
+            }
+            size--;
+            arr = newArr;
+        }
+        return arr[index];
+    }
+
+    @Override
+    public T remove(T value) {
+        int index = 0;
+        for (int i = 0; i < arr.length; i++) {
+            if (value.equals(arr[i])) {
+                index = i;
+                break;
+            }
+        }
+        return remove(index);
+    }
+
+    @Override
+    public int size() {
+        return size;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        return size == 0;
+    }
+
+    @Override
+    public void clear() {
+        arr = null;
+        size = 0;
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder result = new StringBuilder();
+        try {
+            result.append(arr[0].toString());
+            for (int i = 1; i < size; i++) {
+                result.append(", ").append(arr[i]);
+            }
+        } catch (Exception e) {

```

empty catch block is unacceptable